src/Main.java

```diff
@@ -1,76 +1,66 @@
-            System.out.println("На улице " + temprature + " градуса, нужно надеть шапку");
-        } else if(temprature>normal){
-            System.out.println("На улице " + temprature + " градуса можно идти без шапки");
+        System.out.println("Задача 2");
+        for (int i = 10; i > 0; i--) {
+            System.out.println(i);
         }
         //Задача 3
-        int speed=80;
-        int normSpeed=60;
-        if (speed>normSpeed) {
-            System.out.println("Если скорость равна " + speed + " км/ч придётся заплатить штраф");
-        } else if(speed<normSpeed) {
-            System.out.println("Если скорость равна "+speed+" км/ч можно есздить спокойно");
+        System.out.println("Задача 3");
+        for (int i = 0; i < 17; i = i+2) {
+            System.out.println(i);
         }
         //Задача 4
-        int agePeople=22;
-        if (agePeople == 2 || agePeople <= 6) {
-            System.out.println("Если человеку "+agePeople+", то ему нужно ходить в детский сад");
-        } else if(agePeople == 7 || agePeople <= 17) {
-            System.out.println("Если человеку " + agePeople + ", то ему нужно ходить в школу");
-        } else if (agePeople >= 18 || agePeople <= 24) {
-            System.out.println("Если человеку " + agePeople + ", то ему нужно ходить в университет");
-        } else if (agePeople > 24) {
-            System.out.println("Если человеку " + agePeople + ", то ему пора ходить на работу");
+        System.out.println("Задача 4");
+        for (int i = 10; i > -11; i--) {
+            System.out.println(i);
         }
-        //задача 5
-        int kid=17;
-        if(kid<5) {
-            System.out.println("Если возраст ребёнка равен "+kid+" нельзя кататься на атракционе");
-        } else if(kid>5 && kid<14) {
-            System.out.println("Если возраст ребёнка равен "+kid+", то можно кататься в сопровождении взрослого");
-        } else if(kid>14){
-            System.out.println("Если возраст ребёнка равен "+kid+", то можно кататься без сопровождения взрослого");
+        //Задача 5
+        System.out.println("Задача 5");
+        for (int i = 1904; i < 2096; i = i+4) {
+            System.out.println(i + " год является високосным");
         }
         //Задача 6
-        int peoples=80;
-        int train=102;
-        int seatTrain=60;
-        int stayTrain=train-seatTrain;
-        if (peoples>=seatTrain && peoples<=seatTrain) {
-            System.out.println("Если в вагон хотят зайти "+peoples+" людей, то они могут занять сидячее место");
-        } else if(peoples>=stayTrain || peoples<=stayTrain){
-            System.out.println("Если в вагон хотят зайти "+peoples+" людей, то они могут занять сидячие и стоячие места");
-        } else if (peoples>train) {
-            System.out.println("Если в вагон хотят зайти " + peoples + " людей, то не получится, так как вагон будет забит");
+        System.out.println("Задача 6");
+        for (int i = 7; i < 105; i = i+7) {
```

Comment:
```suggestion
        for (int i = 7; i <= 98; i = i+7) {
```
зачем брать какое-то число с потолка, если можно просто идти до нужного?